.travis.yml

```diff
@@ -56,4 +56,7 @@ script:
     - if [ "$DISTRIB" == "ubuntu" ]; then python setup.py nosetests; fi
     - if [ "$DISTRIB" == "conda" ]; then nosetests; fi
     - cd doc
+    # Test a fast run of no-plot
```

Comment:
Comments are switched ... To be honest the kind of comments that I don't find particularly useful.